src/app/mypage/complete_mission/page.tsx

```diff
@@ -1,11 +1,16 @@
-import List from '@/components/ListItem/ListItem';
+import TwoLineListItem from '@/components/ListItem/TwoLineListItem';
```

Comment:
ListItem 에서 index.ts 파일을 만들어서 각각 export 해준다면 이런식으로 끌어다 쓸수있을 것 같아요

```suggestion
import { TwoLineListItem }  from '@/components/ListItem';
```

Reply:
오오 좋아요! 적용하겠습니다 :D